resources/definitions/dagoma_pro_430_base.def.json

```diff
@@ -0,0 +1,207 @@
+{
+    "version": 2,
+    "name": "Dagoma PRO 430 BASE",
+    "inherits": "fdmprinter",
+    "metadata":
+    {
+        "visible": false,
+        "author": "Dagoma",
+        "manufacturer": "Dagoma",
+        "file_formats": "text/x-gcode",
+        "platform": "dagoma_pro_430_base.3mf",
+        "platform_offset": [-125, -79, -130],
+        "has_machine_quality": true,
+        "has_materials": true,
+        "has_variants": true,
+        "variants_name": "Steel Nozzle Size",
+        "preferred_material": "dagoma_generic_pla",
+        "preferred_variant_name": "0.4",
+        "exclude_materials": 
+        [
+            "3D-Fuel_PLA_PRO_Black", 
+            "3D-Fuel_PLA_SnapSupport",
+            "bestfilament_abs_skyblue",
+            "bestfilament_petg_orange",
+            "bestfilament_pla_green",
+            "chromatik_pla",
+            "dsm_arnitel2045_175",
+            "dsm_novamid1070_175",
+            "emotiontech_abs",
+            "emotiontech_absx",
+            "emotiontech_acetate",
+            "emotiontech_asax",
+            "emotiontech_bvoh",
+            "emotiontech_copa",
+            "emotiontech_hips",
+            "emotiontech_nylon_1030",
+            "emotiontech_nylon_1030cf",
+            "emotiontech_nylon_1070",
+            "emotiontech_pc",
+            "emotiontech_pekk",
+            "emotiontech_petg",
+            "emotiontech_pla",
+            "emotiontech_pla_hr_870",
+            "emotiontech_pva-m",
+            "emotiontech_pva-s",
+            "emotiontech_tpu98a",
+            "eryone_petg",
+            "eryone_pla",
+            "eryone_pla_glow",
+            "eryone_pla_matte",
+            "eryone_pla_wood",
+            "eSUN_PETG_Black",
+            "eSUN_PETG_Grey",
+            "eSUN_PETG_Purple",
+            "eSUN_PLA_PRO_Black",
+            "eSUN_PLA_PRO_Grey",
+            "eSUN_PLA_PRO_Purple",
+            "eSUN_PLA_PRO_White",
+            "Extrudr_GreenTECPro_Anthracite_175",
+            "Extrudr_GreenTECPro_Black_175",
+            "Extrudr_GreenTECPro_Blue_175",
+            "Extrudr_GreenTECPro_Nature_175",
+            "Extrudr_GreenTECPro_Red_175",
+            "Extrudr_GreenTECPro_Silver_175",
+            "Extrudr_GreenTECPro_White_175",
+            "fabtotum_abs",
+            "fabtotum_nylon",
+            "fabtotum_pla",
+            "fabtotum_tpu",
+            "fdplast_abs_tomato",
+            "fdplast_petg_gray",
+            "fdplast_pla_olive",
+            "fiberlogy_hd_pla",
+            "filo3d_pla",
+            "filo3d_pla_green",
+            "filo3d_pla_red",
+            "generic_abs",
```

Comment:
Why exclude _all_ generic materials? This might be a bit confusing, since we generally set the profiles to be used with the generic ones. For instance, take a look at the way that UM printers do this. We set the profiles for the generic materials. If the user selects ultimaker PLA, the profiles for generic are used.

The advantage of using this is that if a user does decide to print with a custom material profile, they will still get the default quality profiles.